src/main/java/com/owncloud/android/lib/resources/activities/GetActivitiesRemoteOperation.java

```diff
@@ -102,7 +102,7 @@ public RemoteOperationResult run(NextcloudClient client) {
         String url = client.getBaseUri() + OCS_ROUTE_V12_AND_UP;
 
         // add filter for fileId, if available
-        if (!fileId.isEmpty()) {
+        if (fileId > 0) {
```

Comment:
"unitialized" value is actually ```-1``` so is 0 considered "empty" and thus the ```>0``` ?

Reply:
True, but fileId can never be 0 on server.
-1 is common for uninitialized.
But you are right that this is a bit strange.